docs/benchmarks.md

```diff
@@ -435,9 +435,9 @@ NVBENCH_BENCH(my_cpu_benchmark)
   .set_is_cpu_only(true); // Mark as CPU-only.
 ```
 
-The optional `nvbench::exec_tag::no_gpu` hint may be used to reduce tbe compilation time and
```

Comment:
@alliepiper  but there is still this :/

Reply:
I reset hard the branch to the commit that only fixes this but because PR is closed it doesn't show.